scripts/requirements-base.txt

```diff
@@ -19,6 +19,8 @@ intelhex
 # it's west
 west>=0.7.2
 
+wheel>=0.30.0
```

Comment:
@carlescufi is `wheel` still required

Reply:
I don't think so, try removing it and see if that doesn't break. Otherwise we should include it in our own nrf/scripts/requirements.txt